specification/attributes/discount_handling.md

```diff
@@ -0,0 +1,49 @@
+# Discount Handling
+
+A discount is a pricing construct where providers offer a reduced price for services. Providers may have many types of discounts, including but not limited to commercially negotiated discounts, commitment-based discounts when you agree to a certain amount of usage or spend, and bundled discounts where you receive free or discounted usage of one product or service based on the usage of another. Discount Handling is commonly used in scenarios like verifying discounts were applied and calculating cost savings.
+
+Some discount offers can be purchased from a provider to get reduced prices. The most common example is a commitment-based discount, where you "purchase" a commitment to use or spend a specific amount within a period. There are instances where a portion of the purchased discount may go unused, which results in lost savings and needs to be clearly identifiable at a granular level. To facilitate this, the purchase amount is amortized over the term the discount is applied to (e.g., 1 year).
```

Comment:
I feel like this implies that the reason to amortize is to find the unused portion of a commitment discount which I've not seen in the past. It was more to get a view of cost without the periodic spikes so that the spend is spread based on the period the purchase applies for - which gives you a more accurate view of your spend regardless of the period you're analyzing spend for (allows you to do things like budgeting and forecasting more easily since it's accounting for purchases that were made a long time ago that you're still receiving the benefit of). 

I'm sure @kk09v would have a better accounting / finance explanation that might make sense to list here as well.

Reply:
```suggestion
Some discount offers can be purchased from a provider to get reduced prices. The most common example is a commitment-based discount, where you "purchase" a commitment to use or spend a specific amount within a period. When a commitment isn't fully utilized, the unused amount reduces the potential savings from the discount and can even result in paying higher costs than without the discount. Due to this risk, unused commitment amounts need to be clearly identifiable at a granular level. To facilitate this, unused commitments are recorded with a separate row for each charge period where the commitment was not fully utilized. In order to show the impact of purchased discounts on each discounted row, discount purchases need the purchase amount the be amortized over the term the discount is applied to (e.g., 1 year) with each charge period split and applied to each row that received the discount.
```